<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
background-color: black;
font: 24px;
}


QPushButton {

background-color: yellow;

border-width: 1px;
border-color: yellow;
border-style: solid;

border-radius: 10px;

min-width: 3em;
min-height: 30px;
padding: 6px;



}

QPushButton:pressed {
background-color: rgb(255, 193, 0);
border-color: rgb(255, 193, 0);
}

QPushButton:hover {
border-color: rgb(255, 193, 0);
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frmInitial">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdLocalGame">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Two local players</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdIAGame">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Play against computer</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdNetworkGame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Play on the network</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmAIGame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="cmdLocalBackAI">
          <property name="text">
           <string>←</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdNewAIGame">
          <property name="text">
           <string>↺</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmLocalGame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="cmdLocalBack">
          <property name="text">
           <string>←</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdNewLocalGame">
          <property name="text">
           <string>↺</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdLocalBack</sender>
   <signal>clicked()</signal>
   <receiver>frmLocalGame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLocalBack</sender>
   <signal>clicked()</signal>
   <receiver>frmInitial</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLocalGame</sender>
   <signal>clicked()</signal>
   <receiver>frmInitial</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLocalGame</sender>
   <signal>clicked()</signal>
   <receiver>frmLocalGame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLocalBackAI</sender>
   <signal>clicked()</signal>
   <receiver>frmInitial</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdLocalBackAI</sender>
   <signal>clicked()</signal>
   <receiver>frmAIGame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdIAGame</sender>
   <signal>clicked()</signal>
   <receiver>frmAIGame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdIAGame</sender>
   <signal>clicked()</signal>
   <receiver>frmInitial</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
